src/theory/quantifiers/ce_guided_pbe.cpp

```diff
@@ -606,7 +612,8 @@ void CegConjecturePbe::collectEnumeratorTypes( Node e, TypeNode tn, unsigned enu
             Assert( !et.isNull() );
             d_cinfo[e].d_tinfo[tn].d_strat[cop].d_cenum.push_back( et );
             // need to make this take into account template
-            //Assert( et.getType()==e.getType() || d_einfo[et].getRole()!=enum_io );
+            // Assert( et.getType()==e.getType() ||
+            // d_einfo[et].getRole()!=enum_io );
```

Comment:
Why is this commented out? Remove?

Reply:
I think we get rid of it (I don't remember what this assertion meant)